src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -229,7 +229,7 @@ public function removeCookie($name, $path = '/', $domain = null)
      *
      * @param string $format
      *
-     * @return array
+     * @return Cookie[] An array of cookies
```

Comment:
the description is useless. It repeats exactly the info given by the type `Cookie[]`